Assets/Scripts/UI/MainMenuController.cs

```diff
@@ -26,14 +31,19 @@ public class MainMenuController : MonoBehaviour
     void Start()
     {
         playerInputManagerController = PlayerInputManagerController.Singleton;
-        playerInputManagerController.onPlayerInputJoined += AddPlayer;
         playerInputManagerController.playerInputManager.splitScreen = false;
-
+        playerInputManagerController.onPlayerInputJoined += AddPlayer;
         if (playerInputManagerController.playerInputs.Count > 0)
         {
+            // None of this works... yet
+            //InputSystemUIInputModule inputSystem = GetComponent<InputSystemUIInputModule>();
+            //inputSystem.actionsAsset = playerInputManagerController.playerInputs[0].playerInput.actions;
+            //playerInputManagerController.playerInputs[0].playerInput.uiInputModule = inputSystem;
+            //playerInputManagerController.GetComponent<MultiplayerEventSystem>().playerRoot = defaultMenu;
```

Comment:
Is it necessary to leave this code here? What is the purpose of this?

Reply:
I think it might help to resolve #69, and I wanted to discuss it during today's meeting.

Reply:
I guess it's *fine* to leave it in since this is code that has never been present in the codebase before. Please add a comment that explains this (more than the vague hinting).